actors/miner/src/testing.rs

```diff
@@ -599,12 +599,6 @@ impl PartitionStateSummary {
 struct ExpirationQueueStateSummary {
     pub on_time_sectors: BitField,
     pub early_sectors: BitField,
-    #[allow(dead_code)]
-    pub active_power: PowerPair,
-    #[allow(dead_code)]
-    pub faulty_power: PowerPair,
-    #[allow(dead_code)]
-    pub on_time_pledge: TokenAmount,
```

Comment:
These being dead signals to me that we are missing state invariant checks upstream of this that should use the values. I don't think removing them now is helpful – we should strengthen the invariant checks to use them instead.